plugins/node/opentelemetry-instrumentation-router/src/instrumentation.ts

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
we usually create a file with name from the instrumented library so in this case it would be `router.ts`

Reply:
I'm not sure `router.ts` name would be clearer, we sometimes have `instrumentation` instead so i don't think thats a problem here

Reply:
I'd like to decrease the boilerplate everyone has to attend to when creating new instrumentations. Package-specific file names only increase that. The package name is already in the file path(`opentelemetry-instrumentation-router`) and `package.json`.

Reply:
The whole purpose of having some boilerplate code is that at least it will be unified in some way so any1 new will have less things to learn and the things will be straightforward so If any1 needs to modify something it will be easier. So I see rather a huge benefit in unifying things then leaving this to be different. We usually name the file after the instrumented package, if we instrument more classes in one class then we use name `instrumentation.ts` for example grpc, you can check all of our plugins.

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/479